prow/github/types.go

```diff
@@ -30,14 +30,26 @@ const (
 
 // Possible contents for reactions.
 const (
-	ReactionThumbsUp   = "+1"
-	ReactionThumbsDown = "-1"
-	ReactionLaugh      = "laugh"
-	ReactionConfused   = "confused"
-	ReactionHeart      = "heart"
-	ReactionHooray     = "hooray"
+	ReactionThumbsUp                  = "+1"
+	ReactionThumbsDown                = "-1"
+	ReactionLaugh                     = "laugh"
+	ReactionConfused                  = "confused"
+	ReactionHeart                     = "heart"
+	ReactionHooray                    = "hooray"
+	stateCannotBeChangedMessagePrefix = "state cannot be changed."
 )
 
+// https://developer.github.com/v3/#client-errors
+type ClientError struct {
+	Message string `json:"message"`
+	Errors  []struct {
```

Comment:
this is possibly a bit un-idiomatic but GitHub doesn't even name these and they do have a different """schema""" than the parent type..